src/components/TimeseriesChart/interfaces.ts

```diff
@@ -74,6 +74,10 @@ interface TimeseriesChartPropsBase {
    * Width of the chart
    */
   width?: number;
+  /**
+   * Define y-Axis Subdomains (min, max values for each time series)
+   */
+  ySubDomains?: { [key: number]: [number, number] };
```

Comment:
`key` here is `timeseriesId`, am I right? If yes, it's better to have `timeseriesId` instead of `key`